daml-lf/transaction/src/test/scala/com/digitalasset/daml/lf/value/ValueCoderSpec.scala

```diff
@@ -204,81 +155,45 @@ class ValueCoderSpec
       }
     }
 
-    "do identifier with supported override version" in forAll(idGen, valueVersionGen()) { (i, _) =>
-      val ei = ValueCoder.encodeIdentifier(i)
-      ValueCoder.decodeIdentifier(ei) shouldEqual Right(i)
+    "do identifier with supported override version" in forAll(idGen, transactionVersionGen()) {
+      (i, _) =>
+        val ei = ValueCoder.encodeIdentifier(i)
+        ValueCoder.decodeIdentifier(ei) shouldEqual Right(i)
     }
 
     "do versioned value with supported override version" in forAll(versionedValueGen) {
       case VersionedValue(version, value) => testRoundTripWithVersion(value, version)
     }
-
-    "do versioned value with assigned version" in forAll(valueGen) { v: Value[ContractId] =>
-      testRoundTripWithVersion(v, ValueVersion.assertAssignVersion(v))
-    }
-
-    "versioned value should pass serialization if unsupported override version provided" in
-      forAll(valueGen, unsupportedValueVersionGen) {
-        (value: Value[ContractId], badVer: ValueVersion) =>
-          ValueVersion.acceptedVersions.contains(badVer) shouldBe false
-
-          val actual: proto.VersionedValue = assertRight(
-            ValueCoder
-              .encodeVersionedValueWithCustomVersion(
-                ValueCoder.CidEncoder,
-                VersionedValue(badVer, value),
-              ),
-          )
-
-          actual.getVersion shouldEqual badVer.protoValue
-      }
-
-    "versioned value should fail deserialization if version is not supported" in
-      forAll(valueGen, unsupportedValueVersionGen) {
-        (value: Value[ContractId], badVer: ValueVersion) =>
-          ValueVersion.acceptedVersions.contains(badVer) shouldBe false
-
-          val protoWithUnsupportedVersion: proto.VersionedValue =
-            assertRight(
-              ValueCoder.encodeVersionedValueWithCustomVersion(
-                ValueCoder.CidEncoder,
-                VersionedValue(badVer, value),
-              ),
-            )
-          protoWithUnsupportedVersion.getVersion shouldEqual badVer.protoValue
-
-          val actual: Either[DecodeError, VersionedValue[ContractId]] =
-            ValueCoder.decodeVersionedValue(ValueCoder.CidDecoder, protoWithUnsupportedVersion)
-
-          actual shouldEqual Left(DecodeError(s"Unsupported value version ${badVer.protoValue}"))
-      }
   }
 
-  def testRoundTrip(value: Value[ContractId]): Assertion = {
-    val recovered = ValueCoder.decodeValue(
+  def testRoundTrip(version: TransactionVersion, value: Value[ContractId]): Assertion = {
+    val recovered = ValueCoder decodeValue (
```

Comment:
Oh no, why are we removing the dot here?

Reply:
I do not remember having changed that. I will put it back.